packages/example-getting-started/docs/putting-it-together.md

```diff
@@ -0,0 +1,89 @@
+### Putting it all together
+
+We've got all of our artifacts now, and all that's left is to bind them to our
+[Application](http://loopback.io/doc/en/lb4/Application.html) so that LoopBack's
+[Dependency injection](http://loopback.io/doc/en/lb4/Dependency-injection.html)
+system can tie it all together for us!
+
+LoopBack's
+[boot module](https://github.com/strongloop/loopback-next/tree/master/packages/boot)
+will automatically discover our controllers, repositories, datasources and
+other artifacts and inject them into our application for use.
+
+>**NOTE**: The boot module will only discover and inject artifacts that
+follow our established conventions for artifact directories. Here are some
```

Comment:
This is incorrect. It follows our conventions by default but this config is exposed to the user and they can change it to their own conventions.